tests/publiccloud/slem_basic.pm

```diff
@@ -25,13 +25,42 @@ sub has_wicked {
     return 0;
 }
 
+sub check_avc {
+    my ($self) = @_;
+
+    my $instance = $self->{my_instance};
+    # Read the Access Vector Cache to check for SELinux denials
+    my $avc = $instance->ssh_script_output(cmd => 'sudo ausearch -ts boot -m avc --format raw | grep type=AVC');
+    record_info("AVC at boot", $avc);
+    return if ($avc =~ "no matches");
+
+    ## Gain better formatted logs and upload them for further investigation
+    $instance->ssh_assert_script_run(cmd => 'sudo ausearch -ts boot -m avc > ausearch.txt');
+    assert_script_run("scp " . $instance->username() . "@" . $instance->public_ip . ":ausearch.txt ausearch.txt");
+    upload_logs("ausearch.txt");
+
+    # TODO: Uncomment once all ongoing issues are resolved. For now there will be only a softfailure
+    #die "SELinux access denials on first boot";
```

Comment:
this comment a bit confusing , because if you uncomment this line now test will unconditionally die at this line , I assume this is not intention here

Reply:
Please read the comment in line 42 again.

Reply:
I still don't understand it , but again don't want to have long thread here if you find self-explanatory even after I raise your attention let's keep it . hopefully I will not need to act based just on this comment ...